histomicstk/deeplab/utils/wsi_dataset_util_large_image.py

```diff
@@ -20,6 +20,22 @@
 ################################ main function #################################
 ################################################################################
 
+def get_image(filename):
+    """
+    Read the inut image and perform channel decomposition if necessary.
+    """
+
+    slide = large_image.open(filename)
+    
+    if slide.frames == 3 and slide.bandCount == 1:
+        slide = large_image.open(filename, style={'bands': [{'framedelta': 0, 'palette': '#f00'},{'framedelta': 1, 'palette': '#0f0'},{'framedelta': 2, 'palette': '#00f'}]})
+    elif slide.frames == 1 and slide.bandCount == 3:

```

Comment:
@manthey: I am guessing other image formats such as `tiff, svs, scn` etc. will fall in this category (`slide.frames =1 and slide.bandCount=3`), am I correct as our code needs to be generalized for all image formats we know of?

Reply:
Yes.  large_image abstracts all of the format variations.  This should cover what we expect to work as is; tif there are images that we don't process, we'll have to decide what to do with them when we encounter them.